src/main/webapp/app/shared/layouts/profiles/profile.service.ts

```diff
@@ -26,17 +26,21 @@ export class ProfileService {
                         const data = res.body!;
                         const profileInfo = new ProfileInfo();
                         profileInfo.activeProfiles = data.activeProfiles;
-                        const displayRibbonOnProfiles = data['display-ribbon-on-profiles'].split(',');
 
                         this.mapGuidedTourConfig(data, profileInfo);
                         this.mapAllowedOrionVersions(data, profileInfo);
                         this.mapTestServer(data, profileInfo);
                         ProfileService.mapSaml2Config(data, profileInfo);
 
                         if (profileInfo.activeProfiles) {
-                            const ribbonProfiles = displayRibbonOnProfiles.filter((profile: string) => profileInfo.activeProfiles.includes(profile));
-                            if (ribbonProfiles.length !== 0) {
-                                profileInfo.ribbonEnv = ribbonProfiles[0];
+                            if (data['display-ribbon-on-profiles']) {
```

Comment:
you use `data['display-ribbon-on-profiles']` multiple times, so it would make sense to use const variable here

Reply:
I checked the code and I use `data['display-ribbon-on-profiles']` exactly 2 times right next to each other. So It does not seem to me to be a big issue when it comes to duplication. 

Or is the issue not about duplication, but about a possible change of the value, which could sabotage the execution? This also seems quite unlikely to me, because the two instances are one after another and a quick check, if the value is empty, should be no big/time-consuming issue.

If you disagree, please tell me so and explain to me why this is indeed an issue.